cargo-near/src/abi/generation.rs

```diff
@@ -25,9 +25,10 @@ pub(crate) fn generate_toml(manifest_path: &CargoManifestPath) -> anyhow::Result
         .as_table_mut()
         .expect("[dependencies] is a table specified in the template");
 
-    // Make near-sdk dependency use default features
+    // Make near-sdk dependency not use default features to save on compilation time, but ensure `abi` is enabled
     near_sdk.remove("default-features");
     near_sdk.remove("optional");
+    near_sdk.insert("default-features".to_string(), value::Value::Boolean(false));
```

Comment:
This will be an issue when a crate uses a feature like `unstable`, no?

Reply:
The Cargo.toml I am generating here will be used only for building and running this bit of code: https://github.com/near/cargo-near/blob/main/cargo-near/src/abi/generation.rs#L130-L140.

The reason why I am transforming near-sdk dependency in such a backwards way is that there are so many ways to specify a dependency version that I did not want to deal with (path to a local directory, git URI + branch, git URI + commit hash etc) and instead just deleting stuff that is not needed.